### PR TITLE
Revise Search bar [YEP-69]

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@miltontony @bruskiza tiny review please

File: tuneme/static/css/styles.css [hunk 1]
```diff
@@ -79,7 +79,7 @@ input[type='submit']:focus, a.button:focus {opacity:0.5;}
 
 /* search Black Box */
 #search-form {display:block; clear:both; background:black; padding:15px;}
-    #search-form input {width:79%; background:black; border:1px solid white; color:white;}
+    #search-form input {width:79%; background:black; border:1px solid white; color:white; font:normal 15px 'Quicksilver', Arial, Helvetica, sans-serif;}
```
Comment (miltontony) [inline tuneme/static/css/styles.css]: @saeedmb8 all other fonts we use in this project are called `Quicksand`, why the change?

Comment (smarzban) [inline tuneme/static/css/styles.css]: @miltontony I wanted to use `Quicksand`, but in the ticket says `Quicksilver` should I change it back to `Quicksand`

**Comment by miltontony** [comment]
Otherwise :+1:
